packages/components/src/RadioGroup/RadioGroup.mdx

```diff
@@ -14,7 +14,8 @@ import { useState } from "react";
 
 <ComponentStatus stage="ready" responsive="yes" accessible="yes" />
 
-RadioGroup will allow users to choose from one of a list of provided options.
+RadioGroup will allow users to choose a single selection from one of a list of
```

Comment:
Should we rename this component `InputRadio` or something? Might be good to get ahead of that before this is used in too many places... Just a thought...

Reply:
I like this idea. I think it's only used in one other place in Jobber right now.
I think lets make a followup ticket for that though perhaps? I guess it would be quick to do though…?

Reply:
I also though of this... @seanhealy whats your opinion?

Reply:
Would you be OK doing the rename in a followup PR? As the breaking change will be a _bit_ slower to bring in?